### DISCUSSION TITLE
Cannot filter sites by latitude="null" or latitude=None

### DISCUSSION THREAD

**Discussion by tux-mania** [netbox-community/pynetbox#602] [Q&A] [answered]
Hello all!

I am trying to list all sites which do not have GPS Coordinates.
I initially tried the `is_null=True` filter but it does not work (I then saw these issues talking about it #380 and #442 ).
Unfortunatelly, when trying to use the workaround given `latitude="null"` instead, I get the following error:
```python
pynetbox.core.query.RequestError: The request failed with code 400 Bad Request: {'latitude': ['Enter a number.']}
```

Could you please help me to perform this search?

Thanks!

**Comment by markkuleinio** [comment, marked as answer]
https://docs.netbox.dev/en/stable/reference/filtering/#numeric-fields

```
>>> list(netbox.dcim.sites.filter())
[Site1, Site2]
>>> list(netbox.dcim.sites.filter(latitude__empty=True))
[Site2]
>>> list(netbox.dcim.sites.filter(latitude__empty=False))
[Site1]
>>>
```

**Reply by tux-mania** [reply]
@markkuleinio Thank you very much!!!
I didn't now about the "empty" filter. And thanks for the link to the documentation too!
